resources/views/admin/editProduct.blade.php

```diff
@@ -0,0 +1,153 @@
+                            </div>
+                            <x-error :error="$errors->first('priceForSize')" id="priceForSize" />
+                        </div>
+                        <!-- Select Groups Field -->
+                        @include('partials._select', [
+                            'label' => __('manage-products/products.groups_multi_select_label'),
+                            'placeholder' => __('manage-products/products.groups_multi_select_placeholder'),
+                            'options' => $baseGroups->pluck('name'),
+                            'name' => 'products-group-multiselect',
+                            'selectedGroups' => $chosenGroups->pluck('name')->toArray(),
+                            'class' => __('manage-products/products.groups-multiselect'),
+                        ])
+                        <!-- Product Category Field -->
+                        @include('partials._input', [
+                            'label' => __('manage-products/products.category_input_label'),
+                            'placeholder' => __('manage-products/products.category_input_placeholder'),
+                            'id' => 'product-category',
+                            'type' => 'text',
+                            'name' => 'category',
+                            'value' => $baseChosenCategorie->type,
+                            'disabled' => false,
+                        ])
+                        <!-- Disable Product Checkbox -->
+                        <div class="flex">
+                            @if ($product->inactive === 1)
+                                <input name="inactive-checkbox" type="checkbox"
+                                    class="shrink-0 mt-0.5 border-gray-200 rounded text-blue-600 focus:ring-blue-500 disabled:opacity-50 disabled:pointer-events-none dark:bg-gray-800 dark:border-gray-700 dark:checked:bg-blue-500 dark:checked:border-blue-500 dark:focus:ring-offset-gray-800"
+                                    id="hs-default-checkbox" checked>
+                                <label for="hs-default-checkbox"
+                                    class="text-sm text-gray-500 ms-3 dark:text-gray-400">{{ __('products.active-product') }}</label>
+                            @else
+                                <input name="inactive-checkbox" type="checkbox"
+                                    class="shrink-0 mt-0.5 border-gray-200 rounded text-blue-600 focus:ring-blue-500 disabled:opacity-50 disabled:pointer-events-none dark:bg-gray-800 dark:border-gray-700 dark:checked:bg-blue-500 dark:checked:border-blue-500 dark:focus:ring-offset-gray-800"
+                                    id="hs-default-checkbox">
+                                <label for="hs-default-checkbox"
+                                    class="text-sm text-gray-500 ms-3 dark:text-gray-400">{{ __('products.inactive-product') }}</label>
+                            @endif
```

Comment:
Duplicate code

Reply:
Dit was mijn deel van de code, dus ik zal hierin ook de oplossing delen die ik ook in de PR #82 heb gezet:

```html
<input name="inactive-checkbox" type="checkbox"
                               class="shrink-0 mt-0.5 border-gray-200 rounded text-blue-600 focus:ring-blue-500 disabled:opacity-50 disabled:pointer-events-none dark:bg-gray-800 dark:border-gray-700 dark:checked:bg-blue-500 dark:checked:border-blue-500 dark:focus:ring-offset-gray-800"
                               id="hs-default-checkbox" {{ $product->inactive ? 'checked' : '' }}>
                        <label for="hs-default-checkbox"
                               class="text-sm text-gray-500 ms-3 dark:text-gray-400">
                            {{ $product->inactive ? __('products.active-product') : __('products.inactive-product') }}
                        </label>